lib/knapsack/distributors/report_distributor.rb

```diff
@@ -2,15 +2,15 @@ module Knapsack
   module Distributors
     class ReportDistributor < BaseDistributor
       def sorted_report
-        @sorted_report ||= report.sort_by { |test_path, time| time }.reverse
+        @sorted_report ||= report.sort_by { |_test_path, time| -time }
       end
 
       def sorted_report_with_existing_tests
         @sorted_report_with_existing_tests ||= sorted_report.select { |test_path, time| all_tests.include?(test_path) }
       end
 
       def total_time_execution
-        @total_time_execution ||= sorted_report_with_existing_tests.map(&:last).reduce(0, :+).to_f
+        @total_time_execution ||= sorted_report_with_existing_tests.sum { |_test_path, time| time }.to_f
```

Comment:
I had to change this line because `sum` method in not available in Ruby <= 2.3 and it was [breaking CI build](https://github.com/KnapsackPro/knapsack/runs/2896772003?check_suite_focus=true).

Here is the fix https://github.com/KnapsackPro/knapsack/commit/0be9583ef326efb83e7cd7e3b9bb7a83f00d66b9